src/main/java/storageconnector/S3DBConnector.java

```diff
@@ -281,11 +284,11 @@ public ClientConfig clientConfig_get(ClientID clientID)
 			throw new FBaseStorageConnectorException(e);
 		}
 	}
-	
+
 	@Override
 	public List<ClientID> clientConfig_list() throws FBaseStorageConnectorException {
-		// TODO Auto-generated method stub
-		return null;
+		// TODO 1: IMPLEMENT
+		throw new RuntimeException("NOT YET IMPLEMENTED!");
```

Comment:
advice for the future: There is an UnsupportedOperationException for this exact purpose in Java :)